script/build

```diff
@@ -52,6 +53,7 @@ process.on('unhandledRejection', function (e) {
 checkChromedriverVersion()
 cleanOutputDirectory()
 copyAssets()
+transpilePerPackageTranspilationPaths()
```

Comment:
Similarly to the comment below, how would you feel about renaming this to `transpilePackagesWithCustomTranspiler`?